docs/manual/cluster-admin/how-to-use-alert-system.md

```diff
@@ -248,3 +248,26 @@ Remember to re-build and push the docker image, and restart the `alert-manager`
 ./paictl.py config push -p /cluster-configuration -m service
 ./paictl.py service start -n alert-manager
 ```
+
+## Cluster GPU Utilization Report
+
+We provide the functionality to send cluster GPU utilization report regularly to admin users.
+
+The report includes the statistics for:
+-	Cluster GPU utilization
+-	User GPU utilization 
+-	Job GPU utilization
+
+To enable this feature, you should configure the `alert-manager` field in `services-configuration.yml`.
```

Comment:
Please inform the user which services should be restarted after the config is changed

Reply:
> Please inform the user which services should be restarted after the config is changed

Added